mule-extensions-api/src/main/java/org/mule/runtime/extension/api/connectivity/XATransactionalConnection.java

```diff
@@ -16,7 +16,8 @@
  * @since 1.0
  */
 @MinMuleVersion("4.1")
-public interface XATransactionalConnection extends TransactionalConnection {
+public interface XATransactionalConnection
+    extends TransactionalConnection, org.mule.sdk.api.connectivity.XATransactionalConnection {
```

Comment:
shouldn't we remove the getXAResource and close methods from here, since this interface now extends from the one  in sdk-ap?